governance/program/tests/program_test/mod.rs

```diff
@@ -2011,7 +2014,11 @@ impl GovernanceProgramTest {
 
         self.bench
             .process_transaction(
-                &[create_proposal_transaction],
+                &[
+                    ComputeBudgetInstruction::set_compute_unit_limit(1_000_000u32),
```

Comment:
Let's increase the budget only when it's needed and not every time we invoke `CreateProposal` instruction, it could be generalised at the bench level by adding `with_compute_unit_limit()`which would set a flag indicating next `process_transaction` call should include the budget instructions 

And then please add a test which creates proposal with number of options requiring the budget

Reply:
@SebastianBor Thanks for your feedback.
I added the method as you mentioned to `bench` and provided the test.
Please, review if I haven't missed something.